devices/xiaomi/lumi_sensor_occupy.agl1.json

```diff
@@ -0,0 +1,150 @@
+{
+  "schema": "devcap1.schema.json",
+  "uuid": "72b706a6-2bc0-4a52-ac56-619f7c20ce71",
+  "manufacturername": "aqara",
+  "modelid": "lumi.sensor_occupy.agl1",
+  "vendor": "Xiaomi Aqara",
+  "product": "Presence Sensor FP1E (PS-SE03)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion",
+          "parse": {
+            "fn": "zcl:attr",
+            "ep": "0x01",
+            "cl": "0x0000",
+            "at": "0x0001",
+            "eval": "Item.val = '0.0.0_' + ('0000' + Attr.val.toString()).slice(-4)"
+          },
+          "read": {
+            "fn": "zcl:attr",
+            "ep": "0x01",
+            "cl": "0x0000",
+            "at": "0x0001"
+          },
+          "refresh.interval": 86400
+        },
+        {
+          "name": "attr/swversion_bis",
+          "awake": true,
+          "parse": {
+            "fn": "xiaomi:special",
+            "at": "0x00F7",
+            "idx": "0x08",
+            "eval": "Item.val = '0.0.0_' + ('0000' + (Attr.val & 0xFF).toString()).slice(-4)"
+          },
+          "read": {
+            "fn": "none"
+          }
+        },
```

Comment:
I'd probably go with the special reporting here only. Saves some lines and requests towards the device :)

Reply:
I'm not too sure if the sensor actually sends regular reports for the special attribute.  I've only ever captures a single  report, shortly after resetting the sensor.